datastore/blob/azure_storage_api/__init__.py

```diff
@@ -50,6 +50,37 @@ async def generate_hash(image):
         print(error.__str__())
         raise Exception("Unhandeled Datastore.blob.azure_storage Error")
 
+def build_container_name(name:str,tier:str="user"):
```

Comment:
The spacing on the types does not look right. Are we running ruff on these files?